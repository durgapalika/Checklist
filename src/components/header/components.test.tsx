import { render } from "@testing-library/react";
import Header from ".";

describe("Given Header components", () => {
  describe("StyledText", () => {
    it("Should render correctly", () => {
      const { asFragment } = render(<Header />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
