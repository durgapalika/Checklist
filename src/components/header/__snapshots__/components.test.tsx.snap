// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given Header components StyledText Should render correctly 1`] = `
<DocumentFragment>
  <h3
    class="MuiTypography-root MuiTypography-h3 css-1t3wodr-MuiTypography-root"
    data-testid="headerText"
  >
    My checklist
  </h3>
</DocumentFragment>
`;
